tests/scripts/get_list_for_release_note.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -e
+
+# NOTE: We can skip authenticating github username and token but that will limit the github api request to 60request/IP/hr and by authenticating the limit will be 5000.
+USERNAME=""
+TOKEN=""
+
+# This is just example branch name, change according to uses
+FROM_BRANCH="upstream/release-1.6" # example branch
```

Comment:
will be nice if the script can accept arguments and have help section to understand how it should run

Reply:
added and do we need help section now?

Reply:
done.